packages/react/src/lib/accessibility/Behaviors/Image/imageBehavior.ts

```diff
@@ -8,12 +8,18 @@ import { Accessibility } from '../../types'
  * Adds attribute 'aria-hidden=true', if there is no 'alt' property provided.
  */
 
-const imageBehavior: Accessibility = (props: any) => ({
+const imageBehavior: Accessibility<ImageBehaviorProps> = props => ({
   attributes: {
     root: {
-      'aria-hidden': props['alt'] ? undefined : 'true',
+      'aria-hidden': props.alt || props['aria-label'] ? undefined : 'true',
     },
   },
 })
 
 export default imageBehavior
+
+type ImageBehaviorProps = {
+  /** Alternative text. */
+  alt?: string
+  'aria-label'?: string
```

Comment:
Can we reuse `AccessibilityAttributes`?

```suggestion
  'aria-label'?: AccessibilityAttributes['aria-label'],
```

Reply:
or `& Pick<AccessibilityAttributes, 'aria-label'>`